sympy/printing/pretty/pretty_symbology.py

```diff
@@ -166,6 +166,9 @@ def xstr(*args):
 # latin subscripts
 for l in 'aeioruvx':
     sub[l] = LSUB(l)
+# subscripts from ftp://ftp.unicode.org/Public/6.3.0/ucd/UnicodeData.txt
+for n, l in enumerate('hklmnpst'):
```

Comment:
These new unicode subscript characters are available in unicode version 6.3.0.  Atleast on my install, the `unicodedata` python module is only at unicode 5.2.0.  Your suggestion is clearly the better idea in the long run, but doesn't work right now.  Is there a good way to handle this?